code/game/objects/items/ipc_overloaders.dm

```diff
@@ -338,6 +338,46 @@
 	to_chat(target, SPAN_NOTICE("You begin to feel safer, your surroundings no longer a beehive of potential threats..."))
 
 
+// Unregistered overloaders, may or may not have a virus...
+/obj/item/ipc_overloader/unlabeled
+	name = "basic overloader"
+	desc = "An unlabeled overloader. It still seems functional, but who knows what it would do. What is the worse that could happen?"
```

Comment:
- How do you judge that it "seems functional?"
- "who knows what it could do" would be a question, with a question mark at the end. Since there's another question after, consider using a different phrase altogether.
- "worse that could happen" is incorrect. Use worst.

"An unlabeled overloader. It doesn't look particularly suspicious, but there's no telling its purpose. What's the worst that it could do?"